hnn_core/network.py

```diff
@@ -1340,6 +1415,310 @@ def plot_cells(self, ax=None, show=True):
         """
         return plot_cells(net=self, ax=ax, show=show)
 
+    def write(self, fname, overwrite=True, save_unsimulated=False):
```

Comment:
This is a fair amount of code, I propose moving this to a new file ... `io.py` maybe? You'd basically house all the `write_xxx` functions and `read_xxx` functions there. Then here you'd just call the `write_network` function

Reply:
There would then be a new test file `test_io.py`

Reply:
Sure will do this